src/isotools/_gene_plots.py

```diff
@@ -407,29 +408,42 @@ def gene_track(self, ax=None, title=None, reference=True, select_transcripts=Non
             _ = iter(select_transcripts)
         except TypeError:
             select_transcripts = {self.name: [select_transcripts]}
+    
     contrast = 'white' if np.mean(plt_col.to_rgb(color)) < .5 else 'black'
+    
+    if colorbySqanti:
+        sqanti_palette = {0:{'tag':'FSM', 'color':'#6BAED6'}, 1:{'tag':'ISM', 'color':'#FC8D59'},
+                          2:{'tag':'NIC', 'color':'#78C679'}, 3:{'tag':'NNC', 'color':'#EE6A50'},
+                          4:{'tag':'NOVEL', 'color':'palevioletred'}}
+    
     if ax is None:
         _, ax = plt.subplots(1)
     if x_range is None:
         x_range = (self.start - 100, self.end + 100)
+    
     blocked = []
+    
     if draw_other_genes:
         if isinstance(draw_other_genes, list):
             ol_genes = {self._transcriptome[g] for g in draw_other_genes}.add(self)
         else:
             ol_genes = self._transcriptome.data[self.chrom].overlap(*x_range)
     else:
         ol_genes = {self}
+    
     transcript_list = []
     for g in ol_genes:
         select_tr = g.filter_ref_transcripts(query) if reference else g.filter_transcripts(query, mincoverage, maxcoverage)
+        
         if select_transcripts.get(g.name):
             select_tr = [trid for trid in select_tr if trid in select_transcripts.get(g.name)]
+        
```

Comment:
Personal opinion, but I feel like these two new lines hinder rather than help reading the code